modules/plot/main/coffee/Utils.coffee

```diff
@@ -293,7 +293,7 @@ populateLegendSeries = (selection, series) ->
         .attr('x', 0)
         .attr('width', 10)
         .attr('height', 10)
-        .attr('fill', if s.legendColor then s.legendColor() else s.color) # XXX: the else s.color is there for PieCharts... which should be fixed when pie charts are refactored
+        .attr('fill', if s.legendColor then s.legendColor() else s.fillColor) # XXX: the else s.color is there for PieCharts... which should be fixed when pie charts are refactored
```

Comment:
Fix for pie chart legends - without this they are always black rather than the actual comment.

I am not sure entirely what the comment is referring to re refactoring